easybuild/easyconfigs/o/OpenKIM-Models/OpenKIM-Models-20190221-foss-2018b.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'CMakeMake'
+
+name = 'OpenKIM-Models'
+version = '20190221'
+
+homepage = 'https://openkim.org/'
+description = """Open Knowledgebase of Interatomic Models.
+
+OpenKIM is an API and a collection of interatomic models (potentials) for
+atomistic simulations.  It is a library that can be used by simulation programs
+to get access to the models in the OpenKIM database.
+
+This EasyBuild installs the models.
+ """
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+builddependencies = [
+    ('pkgconfig', '1.3.1', '-Python-3.6.6'),
+]
+
+dependencies = [
+    ('OpenKIM-API', '2.0.1'),
+]
+
+source_urls = ['https://s3.openkim.org/archives/collection/']
+sources = ['OpenKIM-Models-v2-2019-02-21.txz']
+checksums = ['3bd30b0cf2bab314755a66eed621a77c72d3f990818d08366874149be39f208e']
+
+separate_build_dir = True
+configopts = '-DKIM_API_INSTALL_COLLECTION=SYSTEM '
+configopts += '-DCMAKE_C_COMPILER=`which gcc` '
```

Comment:
Not sure how to escape this properly but you should use the environment variables `$CC`, `$CXX`, `$FC` rather than specific compilers, that way the configopts are identical in each toolchain (and less room for copy\paste mistakes)

Reply:
Good point.  I will do that and update.